cabal-testsuite/PackageTests/Regression/T6961/DepInternal/pkg-bar/cabal.out

```diff
@@ -0,0 +1,12 @@
+# cabal v2-build
+Resolving dependencies...
+Build profile: -w ghc-<GHCVER> -O1
+In order, the following will be built:
+ - issue6961-0 (lib:internal-lib) (first run)
+ - issue6961-0 (exe:example) (first run)
+Configuring library 'internal-lib' for issue6961-0..
+Preprocessing library 'internal-lib' for issue6961-0..
+Building library 'internal-lib' for issue6961-0..
+Configuring executable 'example' for issue6961-0..
+Preprocessing executable 'example' for issue6961-0..
+Building executable 'example' for issue6961-0..
```

Comment:
I'm not sure I understand why there are several cabal.out files per test.  Shouldn't this log show a failure, since pkg-bar depends on pkg-foo's internal library?

Reply:
Goot catch. This `caba.out` is just copied here as I made variants of the test, it's not actually a result of test run (see `cabal.out` in a directory one up).

Reply:
I removed these extra files.